pkgs/misc/universal-android-debloater/default.nix

```diff
@@ -0,0 +1,59 @@
+{ pkgs ? import <nixpkgs> { system = builtins.currentSystem; }
```

Comment:
This should not be done in `nixpkgs`. Instead, every required package should be an argument here. That way, they can be overridden if necessary

Reply:
Should be good now